src/Panel/Panel.php

```diff
@@ -89,7 +90,9 @@ public static function areas(): array
         // load plugins
         foreach ($kirby->extensions('areas') as $id => $area) {
             if (is_a($area, 'Closure') === true) {
-                $areas[$id] = static::area($id, (array)$area($kirby));
+                if (isset($areas[$id]) === false) {
```

Comment:
This will prevent replacing areas via plugins

Reply:
Yes, and there's a good reason for that. Right now you could easily ruin the entire Panel with a single plugin, including quite some security relevant stuff. We need to develop a standard way for plugins to replace parts of our main areas without completely fucking everything up.